adapter-cqld4/src/main/resources/activities/baselines/cql-keyvalue-astra.md

```diff
@@ -1,77 +0,0 @@
----
```

Comment:
Was there a reason we wanted to remove these two files? (.md and .yaml)

Reply:
Yes, They were created as specific Astra compatible CQL workloads (That do not create a keyspace). But then you/Jeff had modified all CQL workloads to have an astra specific scenario without the keyspace creation, so this deleted workload is no longer necessary.